os/include/tinyara/os_api_test_drv.h

```diff
@@ -103,21 +103,21 @@ extern "C" {
 #endif
 
 /****************************************************************************
- * Name: kernel_test_drv_register
+ * Name: os_api_test_drv_register
  *
  * Description:
- *   This function creates a device node like "/dev/kernel_test" which will be used
+ *   This function creates a device node like "/dev/os_api_test" which will be used
  *   by the tests that execute kernel side APIs
  *
  *
  ****************************************************************************/
 
-void kernel_test_drv_register(void);
+void os_api_test_drv_register(void);
 
 #undef EXTERN
 #ifdef __cplusplus
 }
 #endif
 
-#endif							/* CONFIG_DRIVERS_KERNEL_TEST */
+#endif							/* CONFIG_DRIVERS_OS_API_TEST */
```

Comment:
__INCLUDE_TINYARA_OS_API_TEST_DRV_H for below #endif comment

Reply:
corrected